pkgs/cronet_http/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.0-wip
```

Comment:
I think that you need to update from master

Reply:
Something that I've missed? The master has `1.1.1` as the latest.

Reply:
Sorry, I was expecting some text from an unmerged PR there. My mistake.